superset-frontend/plugins/plugin-chart-echarts/src/Histogram/buildQuery.ts

```diff
@@ -25,8 +25,10 @@ export default function buildQuery(formData: HistogramFormData) {
   return buildQueryContext(formData, baseQueryObject => [
     {
       ...baseQueryObject,
+      extras: { where: `${column} IS NOT NULL` },
```

Comment:
Given that `column` can only contain numeric values, we automatically exclude `NULL` values from the result. This also aligns better with the limit control.